### ISSUE TITLE
Nortek file processing - Use memory mapped file?

### ISSUE THREAD

**Issue by NickMortimer** [aodn/imos-toolbox#435]
Hi G,

Had a quick look at the new readbinary for nortek and just thought you might like to use the memmapped file function.

https://au.mathworks.com/help/matlab/ref/memmapfile.html

**Comment by ggalibert** [comment]
Thanks Nick, had thought about it and would definitely be a great solution for solving some memory issues. There is a good example here on how to implement memory mapping: https://au.mathworks.com/help/matlab/import_export/map-a-file-to-memory.html although the Nortek binary file format is not static and can include various type of records anywhere in the file.

**Comment by NickMortimer** [comment]
Yep my C++ Nortek parser just opens the file using memap and then casts the whole file to byte because of the unknown record types
